doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,39 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Storage and file systems:
+
+  * Fixed a possible NULL pointer dereference when using any of the ``fs_`` functions.
+    The functions will now return an error code in this case.
+  * Fixed a garbage-collection issue in the NVS subsystem.
+
+* Devicetree:
+
+  * Removed all nRF-specific aliases to particular hardware peripherals, since they are no longer needed now that nodes are addressable by node label. As an example, you should now use ``DT_NODELABLE(i2c0)`` instead of ``DT_ALIAS(i2c_0)``.
```

Comment:
```suggestion
  * Removed all nRF-specific aliases to particular hardware peripherals, because they are no longer needed now that nodes can be addressed by node labels. 
    For example, you should now use ``DT_NODELABLE(i2c0)`` instead of ``DT_ALIAS(i2c_0)``.
```
And it's probably NODELABEL and not LABLE?